src/router.rs

```diff
@@ -0,0 +1,8 @@
+use actix_web::web::{get, resource, ServiceConfig};
+
+use crate::handlers::hello;
+
+/// Registers HTTP services to the current server instance
+pub fn router(cfg: &mut ServiceConfig) {
+    cfg.service(resource("/hello").route(get().to(hello)));
+}
```

Comment:
`router` is in charge of assigning a `handler` to a `resource` (or path).
This structure will help us to prefix multiple handlers on a DRY manner, having a router defined like these also help us navigating through the project and reaching to endpoints implementations with ease.